plugins/module_utils/pulp.py

```diff
@@ -213,11 +213,15 @@ def delete(self):
         self.entity = None
         self.module.set_changed()
 
-    def sync(self, remote_href):
+    def sync(self, remote_href, mirror):
         if not hasattr(self, "_sync_id"):
             raise SqueezerException("This entity is not syncable.")
+
+        body = {"remote": remote_href}
+        if mirror:
+            body["mirror"] = mirror
```

Comment:
```suggestion
        if mirror is not None:
            body["mirror"] = mirror
```
If we also send mirror=False, we can overwrite (on demand) whatever the server sets as default.